docker-compose.yaml

```diff
@@ -5,28 +5,36 @@ services:
       context: .
       dockerfile: Dockerfile
     environment:
-      - SERVERFULL_RUNTIME_HTTPSERVER_ADDRESS=:8081
```

Comment:
I think a lot of our docker-compose.yml files in our repos have this format, if it's wrong, I think the other repos should be fixed as well. I couldn't find docs showing that one particular way was right or wrong though, can you share the doc if you found it?

Reply:
It's not wrong, both formats are acceptable in the spec. I find this format simpler to troubleshoot when there are issues with the environment variables using YAML linters and syntax highlighting. The other list format treats both the key and value as a single string, so it's a little harder for me to detect problems. If you've got a strong preference, I'll revert it back.

Reply:
I don't have a preference for the formatting, but I would prefer if things like this be consistent across our repos. I don't want this to be a blocker for merging. Maybe we could have a separate conversation about this.

Reply:
Sure, I'd be happy to.